rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/controllers/v3/DatasetControllerV3IntegrationSuite.scala

```diff
@@ -73,7 +73,7 @@ class DatasetControllerV3IntegrationSuite extends BaseRestApiTestV3 with BeforeA
         val response = sendPost[Dataset, Validation](apiUrl, bodyOpt = Some(dataset))
         assertCreated(response)
         response.getBody shouldBe Validation.empty.withWarning("keyD", "Property 'keyD' is recommended to be present, but was not found!")
-        val locationHeader = response.getHeaders.getFirst("location")
+        val locationHeader = response.getHeaders.getFirst("Location")
```

Comment:
How did this work before? Or is the function case insensitive and just convention dictates using capital `L`?

Reply:
The latter exactly:
> function case insensitive and just convention dictates using capital `L`